### PR TITLE
Add a new language: Arabic (5/6 tutorials)

### PR DESCRIPTION
**Pull request by EyadMahm0ud**
I added the Arabic language to five tutorials:
- joystick.json
- healthBar.json
- cameraParallax.json
- timer.json
- plinkoMultiplier.json

I have reviewed the translations well, So it's not supposed to be broken, or corrupted Thai translations, but if it is, I'll try to fix it.
For the flingGame.json, it will be translated in the next three days or this week.

### CODE REVIEW

File: tutorials/in-app/healthBar.json [hunk 1]
```diff
@@ -303,7 +320,8 @@
             "fr": "Changeons son nom en **Interface**.",
             "es": "Cambiamos su nombre a **Interface**.",
             "pt": "Mudamos o nome para **Interface**.",
-            "th": "เปลี่ยนชื่อเป็น **Interface**"
+            "th": "เปลี่ยนชื่อเป็น **Interface**",
+			"ar": "تغيير الاسم إلى **Interface**."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:324]:
```suggestion
            "th": "เปลี่ยนชื่อเป็น **Interface**",
            "ar": "تغيير الاسم إلى **Interface**."
```

File: tutorials/in-app/healthBar.json [hunk 2]
```diff
@@ -339,7 +358,8 @@
             "fr": "Ouvrez le panneau des **objets**.",
             "es": "Abre el panel de **objetos**.",
             "pt": "Abra o painel de **objetos**.",
-            "th": "เปิดแผงควบคุม **วัตถุ**"
+            "th": "เปิดแผงควบคุม **วัตถุ**",
+			"ar": "فتح لوحة **الكائنات**."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:362]:
```suggestion
            "th": "เปิดแผงควบคุม **วัตถุ**",
            "ar": "فتح لوحة **الكائنات**."
```

File: tutorials/in-app/healthBar.json [hunk 3]
```diff
@@ -358,7 +378,8 @@
             "fr": "Faites glisser **$(resourceBar)** du menu à la scène.",
             "es": "Arrastra **$(resourceBar)** desde el menú a la escena.",
             "pt": "Arraste **$(resourceBar)** do menu para a cena.",
-            "th": "ลาก **$(resourceBar)** ใส่ลงใน scene"
+            "th": "ลาก **$(resourceBar)** ใส่ลงใน scene",
+			"ar": "سحب الـ **$(resourceBar)** إلى المشهد."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:382]:
```suggestion
            "th": "ลาก **$(resourceBar)** ใส่ลงใน scene",
            "ar": "سحب الـ **$(resourceBar)** إلى المشهد."
```

File: tutorials/in-app/healthBar.json [hunk 4]
```diff
@@ -367,7 +388,8 @@
             "fr": "Sélectionnez **$(resourceBar)**, puis faites-le glisser à la scène.",
             "es": "Selecciona **$(resourceBar)**, luego arrástralo a la escena.",
             "pt": "Selecione **$(resourceBar)**, em seguida, arraste-o para a cena.",
-            "th": "เลือก **$(resourceBar)** แล้วลากมันเข้า scene"
+            "th": "เลือก **$(resourceBar)** แล้วลากมันเข้า scene",
+			"ar": "تحديد الـ  **$(resourceBar)** ثم سحبه إلى المشهد."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:392]:
```suggestion
            "th": "เลือก **$(resourceBar)** แล้วลากมันเข้า scene",
            "ar": "تحديد الـ  **$(resourceBar)** ثم سحبه إلى المشهد."
```

File: tutorials/in-app/healthBar.json [hunk 5]
```diff
@@ -380,7 +402,8 @@
             "fr": "J'ai terminé",
             "es": "He terminado",
             "pt": "Terminei",
-            "th": "เสร็จแล้ว"
+            "th": "เสร็จแล้ว",
+			"ar": "انتهيت"
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:406]:
```suggestion
            "th": "เสร็จแล้ว",
            "ar": "انتهيت"
```

File: tutorials/in-app/healthBar.json [hunk 6]
```diff
@@ -392,7 +415,8 @@
...
             "es": "¿Ves el **rectángulo negro** en el centro de la escena? Esta es la **vista de la cámara**. Es el punto de vista desde el que el jugador verá el juego.\n\nColoca $(resourceBar) en la esquina superior izquierda del rectángulo.",
             "pt": "Você vê o **retângulo preto** no meio da cena? Esta é a **visão da câmera**. É o ponto de vista a partir do qual o jogador verá o jogo.\n\nColoque $(resourceBar) no canto superior esquerdo do retângulo.",
-            "th": "เห็น **สี่เหลี่ยมสีดำ** ตรงกลางของ scene ไหม? นั่นคือ**มุมมองกล้อง** เป็นมุมมองที่ผู้เล่นจะมองเห็นเกม\n\nจัดวาง $(resourceBar) ให้อยู่ในมุมซ้ายบนของสี่เหลี่ยม\n\nเมื่อเสร็จแล้ว คลิกด้านล่าง"
+            "th": "เห็น **สี่เหลี่ยมสีดำ** ตรงกลางของ scene ไหม? นั่นคือ**มุมมองกล้อง** เป็นมุมมองที่ผู้เล่นจะมองเห็นเกม\n\nจัดวาง $(resourceBar) ให้อยู่ในมุมซ้ายบนของสี่เหลี่ยม\n\nเมื่อเสร็จแล้ว คลิกด้านล่าง",
+			"ar": "هل ترى **الإطار المستطيلي الأسود** في منتصف المشهد؟ ها هو **رؤية الكاميرا**. إنه يحيط بجزء من اللعبة الذي يمكن للاعبين رؤيته.\n\nوضع الـ $(resourceBar) في أعلى يسار زاوية **رؤية الكاميرا**."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:419]:
```suggestion
            "th": "เห็น **สี่เหลี่ยมสีดำ** ตรงกลางของ scene ไหม? นั่นคือ**มุมมองกล้อง** เป็นมุมมองที่ผู้เล่นจะมองเห็นเกม\n\nจัดวาง $(resourceBar) ให้อยู่ในมุมซ้ายบนของสี่เหลี่ยม\n\nเมื่อเสร็จแล้ว คลิกด้านล่าง",
            "ar": "هل ترى **الإطار المستطيلي الأسود** في منتصف المشهد؟ ها هو **رؤية الكاميرا**. إنه يحيط بجزء من اللعبة الذي يمكن للاعبين رؤيته.\n\nوضع الـ $(resourceBar) في أعلى يسار زاوية **رؤية الكاميرا**."
```

File: tutorials/in-app/healthBar.json [hunk 7]
```diff
@@ -413,7 +437,8 @@
             "fr": "Maintenant, branchons la vie du **$(player)** avec **$(resourceBar)** ! Cliquez sur l'onglet Évènements de la scène $(level) pour accéder à la **feuille d'événements**.",
             "es": "Ahora, conectemos la vida del **$(player)** con **$(resourceBar)**! Abramos la **hoja de eventos** de la escena $(level).",
             "pt": "Agora, vamos conectar a vida do **$(player)** com **$(resourceBar)**! Vamos **abrir a Folha de Eventos** da cena $(level).",
-            "th": "ทีนี้ มาเชื่อมเลือดของ **$(player)** กับ **$(resourceBar)**กันเถอะ! ให้ **เปิดชี้ทอีเวนต์** จาก scene $(level)"
+            "th": "ทีนี้ มาเชื่อมเลือดของ **$(player)** กับ **$(resourceBar)**กันเถอะ! ให้ **เปิดชี้ทอีเวนต์** จาก scene $(level)",
+			"ar": "الآن سوف نربط صحة الـ **$(player)** بالـ **$(resourceBar)**! الضغط على تبويب الأحداث للذهاب إلى **صفحة الأحداث** الخاصة بالمشهد $(level)."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:441]:
```suggestion
            "th": "ทีนี้ มาเชื่อมเลือดของ **$(player)** กับ **$(resourceBar)**กันเถอะ! ให้ **เปิดชี้ทอีเวนต์** จาก scene $(level)",
            "ar": "الآن سوف نربط صحة الـ **$(player)** بالـ **$(resourceBar)**! الضغط على تبويب الأحداث للذهاب إلى **صفحة الأحداث** الخاصة بالمشهد $(level)."
```

File: tutorials/in-app/healthBar.json [hunk 8]
```diff
@@ -432,7 +457,8 @@
             "fr": "**Ajoutons une action** pour changer le nombre de vies affichées par $(resourceBar).",
             "es": "**Agreguemos una acción** para cambiar el valor de relleno de $(resourceBar).",
             "pt": "**Adicionemos uma ação** para alterar o valor de preenchimento de $(resourceBar).",
-            "th": "ให้ **เพิ่มการกระทำ** เพื่อเปลี่ยนค่าที่จะเพิ่มขึ้นของ $(resourceBar)"
+            "th": "ให้ **เพิ่มการกระทำ** เพื่อเปลี่ยนค่าที่จะเพิ่มขึ้นของ $(resourceBar)",
+			"ar": "دعنا نقوم بـ**إضافة إجراء** لتعريف عدد المحاولات التي ستُعرض على $(resourceBar)."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:461]:
```suggestion
            "th": "ให้ **เพิ่มการกระทำ** เพื่อเปลี่ยนค่าที่จะเพิ่มขึ้นของ $(resourceBar)",
            "ar": "دعنا نقوم بـ**إضافة إجراء** لتعريف عدد المحاولات التي ستُعرض على $(resourceBar)."
```

File: tutorials/in-app/healthBar.json [hunk 9]
```diff
@@ -449,7 +475,8 @@
             "fr": "Sélectionnez **$(resourceBar)**.",
             "es": "Seleccione **$(resourceBar)**.",
             "pt": "Selecione **$(resourceBar)**.",
-            "th": "เลือก **$(resourceBar)**"
+            "th": "เลือก **$(resourceBar)**",
+			"ar": "تحديد **$(resourceBar)**."
```
Comment (Bouh) [inline tutorials/in-app/healthBar.json:479]:
```suggestion
            "th": "เลือก **$(resourceBar)**",
            "ar": "تحديد **$(resourceBar)**."
```

File: tutorials/in-app/joystick.json [hunk 10]
```diff
@@ -508,7 +538,8 @@
             "fr": "Bien, nous avons terminé ! Testons notre jeu pour voir les changements que nous avons apportés ! Cliquez sur le bouton **Aperçu**.",
             "es": "¡Bien, hemos terminado! ¡Probemos nuestro juego para ver los cambios que hemos hecho! Haga clic en el botón **Vista previa**.",
             "pt": "Bem, terminamos! Vamos testar nosso jogo para ver as mudanças que fizemos! Clique no botão **Visualizar**.",
-            "th": "เอาล่ะ เสร็จเรียบร้อยแล้ว! มาทดสอบเกมของเราเพื่อดูการเปลี่ยนแปลงที่เราทำกันดีกว่า! คลิกที่ปุ่ม **ดูตัวอย่าง**"
+            "th": "เอาล่ะ เสร็จเรียบร้อยแล้ว! มาทดสอบเกมของเราเพื่อดูการเปลี่ยนแปลงที่เราทำกันดีกว่า! คลิกที่ปุ่ม **ดูตัวอย่าง**",
+			"ar": "حسنًا، لقد انتهينا! هيّا نختبر لعبتنا لنرى التغييرات التي قمنا بها! الضغط على زر **معاينة**."
```
Comment (Bouh) [inline tutorials/in-app/joystick.json:542]:
```suggestion
            "th": "เอาล่ะ เสร็จเรียบร้อยแล้ว! มาทดสอบเกมของเราเพื่อดูการเปลี่ยนแปลงที่เราทำกันดีกว่า! คลิกที่ปุ่ม **ดูตัวอย่าง**",
            "ar": "حسنًا، لقد انتهينا! هيّا نختبر لعبتنا لنرى التغييرات التي قمنا بها! الضغط على زر **معاينة**."
```

File: tutorials/in-app/joystick.json [hunk 11]
```diff
@@ -491,7 +520,8 @@
             "fr": "C'est tout ! Maintenant, la manette contrôlera le vaisseau !",
             "es": "¡Eso es todo! ¡Ahora el joystick controlará el barco!",
             "pt": "É isso aí! Agora o joystick controlará o navio!",
-            "th": "นั่นเอง! ตอนนี้ joystick จะควบคุมเรือ!"
+            "th": "นั่นเอง! ตอนนี้ joystick จะควบคุมเรือ!",
+			"ar": "هذا كل شيء! الآن ستتحكم عصا التحكم بالسفينة الفضائية!"
```
Comment (Bouh) [inline tutorials/in-app/joystick.json:524]:
```suggestion
            "th": "นั่นเอง! ตอนนี้ joystick จะควบคุมเรือ!",
            "ar": "هذا كل شيء! الآن ستتحكم عصا التحكم بالسفينة الفضائية!"
```

File: tutorials/in-app/joystick.json [hunk 12]
```diff
@@ -473,7 +501,8 @@
             "fr": "Changeons le mode de la manette, pour qu'elle puisse être contrôlée à **360°**.",
             "es": "Actualicemos el modo de la palanca, para que pueda ser controlada a **360°**.",
             "pt": "Vamos atualizar o modo do joystick, para que ele possa ser controlado a **360°**.",
-            "th": "อัปเดตโหมดของ joystick ให้สามารถควบคุมได้ทั้ง **360°**"
+            "th": "อัปเดตโหมดของ joystick ให้สามารถควบคุมได้ทั้ง **360°**",
+			"ar": "دعنا نقوم بتحديث وضع عصا التحكم حتى يمكن التحكم به في وضع **360°**"
```
Comment (Bouh) [inline tutorials/in-app/joystick.json:505]:
```suggestion
            "th": "อัปเดตโหมดของ joystick ให้สามารถควบคุมได้ทั้ง **360°**",
            "ar": "دعنا نقوم بتحديث وضع عصا التحكم حتى يمكن التحكم به في وضع **360°**"
```

File: tutorials/in-app/joystick.json [hunk 13]
```diff
@@ -455,7 +482,8 @@
             "fr": "Sélectionnez le comportement **Top down multi-touch controller mapper**.",
             "es": "Selecciona el comportamiento **Top down multi-touch controller mapper**.",
             "pt": "Selecione o comportamento **Top down multi-touch controller mapper**.",
-            "th": "เลือกพฤติกรรม **Top down multi-touch controller mapper**"
+            "th": "เลือกพฤติกรรม **Top down multi-touch controller mapper**",
+			"ar": "تحديد السلوك **Top down multi-touch controller mapper**."
```
Comment (Bouh) [inline tutorials/in-app/joystick.json:486]:
```suggestion
            "th": "เลือกพฤติกรรม **Top down multi-touch controller mapper**",
            "ar": "تحديد السلوك **Top down multi-touch controller mapper**."
```

File: tutorials/in-app/joystick.json [hunk 14]
```diff
@@ -437,7 +463,8 @@
             "fr": "Ajoutons le comportement **Top down multi-touch controller mapper** qui a été installé.",
             "es": "Agreguemos el comportamiento **Top down multi-touch controller mapper** que se instaló.",
             "pt": "Vamos adicionar o comportamento **Top down multi-touch controller mapper** que foi instalado.",
-            "th": "เพิ่ม **Top down multi-touch controller mapper** พฤติกรรมที่ติดตั้งไว้"
+            "th": "เพิ่ม **Top down multi-touch controller mapper** พฤติกรรมที่ติดตั้งไว้",
+			"ar": "دعنا نقوم بإضافة السلوك **Top down multi-touch controller mapper** الذي تم تثبيته."
```
Comment (Bouh) [inline tutorials/in-app/joystick.json:467]:
```suggestion
            "th": "เพิ่ม **Top down multi-touch controller mapper** พฤติกรรมที่ติดตั้งไว้",
            "ar": "دعنا نقوم بإضافة السلوك **Top down multi-touch controller mapper** الذي تم تثبيته."
```

File: tutorials/in-app/joystick.json [hunk 15]
```diff
@@ -417,7 +442,8 @@
             "fr": "Les **comportements** de **l'objet** se trouvent dans cet onglet.",
             "es": "Los **comportamientos** del **objeto** se encuentran en esta pestaña.",
             "pt": "Os **comportamentos** do **objeto** estão nesta guia.",
-            "th": "ดู **พฤติกรรม** ของ **วัตถุ** ที่นี่"
+            "th": "ดู **พฤติกรรม** ของ **วัตถุ** ที่นี่",
+			"ar": "رؤية **سلوكيات الكائن** تأتي من هنا."
```
Comment (Bouh) [inline tutorials/in-app/joystick.json:446]:
```suggestion
            "th": "ดู **พฤติกรรม** ของ **วัตถุ** ที่นี่",
            "ar": "رؤية **سلوكيات الكائن** تأتي من هنا."
```

File: tutorials/in-app/joystick.json [hunk 16]
```diff
@@ -400,7 +424,8 @@
...
             "es": "Vinculemos nuestro $(joystick) y nuestro $(ship) gracias a los **comportamientos** que vinieron con el prefab:\n\nSelecciona, luego presiona por un largo tiempo en **$(ship)**, luego selecciona **Editar comportamientos**.",
             "pt": "Vamos vincular nosso $(joystick) e nosso $(ship) graças aos **comportamentos** que vieram com o prefab:\n\nSelecione, depois pressione por um longo tempo em **$(ship)**, depois selecione **Editar comportamentos**.",
-            "th": "ให้เชื่อมโยง $(joystick) และ $(ship) ของเราด้วย **พฤติกรรม** ที่มาพร้อมกับ prefab:\n\nเลือก แล้วกดค้างที่ **$(ship)** แล้วเลือก **แก้ไขพฤติกรรม**"
+            "th": "ให้เชื่อมโยง $(joystick) และ $(ship) ของเราด้วย **พฤติกรรม** ที่มาพร้อมกับ prefab:\n\nเลือก แล้วกดค้างที่ **$(ship)** แล้วเลือก **แก้ไขพฤติกรรม**",
+			"ar": "دعنا نربط  $(joystick) و$(ship) الخاصان بنا بواسطة **السلوكيات** التي أتت مع الكائن المجهز مسبقًا: \n\nتحديد، ثم ضغطة مطولة على الـ **$(ship)** وتحديد **تحرير السلوكيات**."
```
Comment (Bouh) [inline tutorials/in-app/joystick.json:428]:
```suggestion
            "th": "ให้เชื่อมโยง $(joystick) และ $(ship) ของเราด้วย **พฤติกรรม** ที่มาพร้อมกับ prefab:\n\nเลือก แล้วกดค้างที่ **$(ship)** แล้วเลือก **แก้ไขพฤติกรรม**",
            "ar": "دعنا نربط  $(joystick) و$(ship) الخاصان بنا بواسطة **السلوكيات** التي أتت مع الكائن المجهز مسبقًا: \n\nتحديد، ثم ضغطة مطولة على الـ **$(ship)** وتحديد **تحرير السلوكيات**."
```

File: tutorials/in-app/cameraParallax.json [hunk 17]
```diff
@@ -44,7 +46,8 @@
           "fr": "- Comment créer un calque\n\n- Comment ajouter une extension\n\n- Comment contrôler la caméra\n\n- Comment faire bouger des Tiled Sprites à différentes vitesses",
           "es": "- Cómo crear una capa\n\n- Cómo agregar una extensión\n\n- Cómo controlar la cámara\n\n- Cómo hacer que los Tiled Sprites se muevan a diferentes velocidades",
           "pt": "- Como criar uma camada\n\n- Como adicionar uma extensão\n\n- Como controlar a câmera\n\n- Como fazer Tiled Sprites se movem a diferentes velocidades",
-          "th": "- วิธีสร้างเลเยอร์\n\n- วิธีเพิ่ม extension\n\n- วิธีควบคุมกล้อง\n\n- วิธีทำให้ Tiled Sprites เคลื่อนที่ด้วยค่าความเร็วต่างๆ"
+          "th": "- วิธีสร้างเลเยอร์\n\n- วิธีเพิ่ม extension\n\n- วิธีควบคุมกล้อง\n\n- วิธีทำให้ Tiled Sprites เคลื่อนที่ด้วยค่าความเร็วต่างๆ",
+		  "ar": "- إنشاء طبقة\n\n- إضافة ملحق\n\n- التحكم في الكاميرا\n\n- إنشاء كائنات مبلطة تتحرك بسرعات مختلفة"
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:50]:
```suggestion
          "th": "- วิธีสร้างเลเยอร์\n\n- วิธีเพิ่ม extension\n\n- วิธีควบคุมกล้อง\n\n- วิธีทำให้ Tiled Sprites เคลื่อนที่ด้วยค่าความเร็วต่างๆ",
          "ar": "- إنشاء طبقة\n\n- إضافة ملحق\n\n- التحكم في الكاميرا\n\n- إنشاء كائنات مبلطة تتحرك بسرعات مختلفة"
```

File: tutorials/in-app/cameraParallax.json [hunk 18]
```diff
@@ -53,7 +56,8 @@
           "fr": "Vous pouvez continuer à ajouter des choses à ce jeu ou le publier !",
           "es": "¡Puedes seguir agregando cosas a este juego o publicarlo !",
           "pt": "Você pode continuar adicionando coisas a este jogo ou publicá-lo!",
-          "th": "คุณสามารถพัฒนาเกมนี้ต่อไปหรือจะเผยแพร่เลยก็ได้!"
+          "th": "คุณสามารถพัฒนาเกมนี้ต่อไปหรือจะเผยแพร่เลยก็ได้!",
+		  "ar": "يمكنك الاستمرار في إضافة أشياء إلى هذه اللعبة أو نشرها!"
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:60]:
```suggestion
          "th": "คุณสามารถพัฒนาเกมนี้ต่อไปหรือจะเผยแพร่เลยก็ได้!",
          "ar": "يمكنك الاستمرار في إضافة أشياء إلى هذه اللعبة أو نشرها!"
```

File: tutorials/in-app/cameraParallax.json [hunk 19]
```diff
@@ -72,7 +76,8 @@
             "fr": "Améliorons ce jeu de plateforme en suivant le personnage avec la caméra !",
             "es": "¡Mejoremos este juego de plataforma siguiendo al personaje con la cámara!",
             "pt": "Melhore este jogo de plataforma seguindo o personagem com a câmera!",
-            "th": "มาทำให้เกม platformer เกมนี้ดีขึ้นจากการติดตามตัวละครด้วยกล้องกันเถอะ!"
+            "th": "มาทำให้เกม platformer เกมนี้ดีขึ้นจากการติดตามตัวละครด้วยกล้องกันเถอะ",
+			"ar": "دعنا نحسن لعبة المنصات هذه بمتابعة الشخصية بواسطة الكاميرا!"
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:80]:
```suggestion
            "th": "มาทำให้เกม platformer เกมนี้ดีขึ้นจากการติดตามตัวละครด้วยกล้องกันเถอะ",
          "ar": "دعنا نحسن لعبة المنصات هذه بمتابعة الشخصية بواسطة الكاميرا!"
```

File: tutorials/in-app/cameraParallax.json [hunk 20]
```diff
@@ -81,7 +86,8 @@
             "fr": "Pour cela, nous allons installer une **extension**.\n\nOuvrez le **Gestionnaire de projet**.",
             "es": "Para esto, instalaremos una **extensión**.\n\nAbre el **Administrador de proyectos**.",
             "pt": "Para isso, instalaremos uma **extensão**.\n\nAbra o **Gerenciador de projetos**.",
-            "th": "โดยเราจะติดตั้ง **extension**.\n\nเปิด **โปรเจกต์เมเนเจอร์**"
+            "th": "โดยเราจะติดตั้ง **extension**.\n\nเปิด **โปรเจกต์เมเนเจอร์**",
+			"ar": "لفعل هذا، سوف نثبت **ملحقًا**.\n\nفتح **مدير المشروع**."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:90]:
```suggestion
            "th": "โดยเราจะติดตั้ง **extension**.\n\nเปิด **โปรเจกต์เมเนเจอร์**",
            "ar": "لفعل هذا، سوف نثبت **ملحقًا**.\n\nفتح **مدير المشروع**."
```

File: tutorials/in-app/cameraParallax.json (hunk 19 above)
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:80]:
```suggestion
            "th": "มาทำให้เกม platformer เกมนี้ดีขึ้นจากการติดตามตัวละครด้วยกล้องกันเถอะ",
            "ar": "دعنا نحسن لعبة المنصات هذه بمتابعة الشخصية بواسطة الكاميرا!"
```

File: tutorials/in-app/cameraParallax.json [hunk 21]
```diff
@@ -98,7 +104,8 @@
             "fr": "Ouvrez l'onglet **Extensions**.",
             "es": "Abre la pestaña **Extensiones**.",
             "pt": "Abra a guia **Extensões**.",
-            "th": "เปิด **แท็บ Extensions**"
+            "th": "เปิด **แท็บ Extensions**",
+			"ar": "فتح تبويب **الملحقات**."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:108]:
```suggestion
            "th": "เปิด **แท็บ Extensions**",
            "ar": "فتح تبويب **الملحقات**."
```

File: tutorials/in-app/cameraParallax.json [hunk 22]
```diff
@@ -117,7 +124,8 @@
             "fr": "Ouvrez le **Magasin d'extension**.",
             "es": "Abre la **tienda de extensiones**",
             "pt": "Abra a **loja de extensões**",
-            "th": "เปิด **ร้านค้า extension**"
+            "th": "เปิด **ร้านค้า extension**",
+			"ar": "فتح متجر الملحقات."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:128]:
```suggestion
            "th": "เปิด **ร้านค้า extension**",
            "ar": "فتح متجر الملحقات."
```

File: tutorials/in-app/cameraParallax.json [hunk 23]
```diff
@@ -136,7 +144,8 @@
             "fr": "Cherchez l'extension **smooth camera**.",
             "es": "Busca la extensión **smooth camera**.",
             "pt": "Procure pela extensão **smooth camera**.",
-            "th": "ค้นหา extension **smooth camera**"
+            "th": "ค้นหา extension **smooth camera**",
+			"ar": "البحث عن الملحق **smooth camera*.*"
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:148]:
```suggestion
            "th": "ค้นหา extension **smooth camera**",
            "ar": "البحث عن الملحق **smooth camera*.*"
```

File: tutorials/in-app/cameraParallax.json [hunk 24]
```diff
@@ -155,7 +164,8 @@
             "fr": "Cliquez sur l'extension **Smooth Camera**.",
             "es": "Haz clic en la extensión **Smooth Camera**.",
             "pt": "Clique na extensão **Smooth Camera**.",
-            "th": "คลิกที่ extension **Smooth Camera**"
+            "th": "คลิกที่ extension **Smooth Camera**",
+			"ar": "الضغط على الملحق **smooth camera*.*"
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:168]:
```suggestion
            "th": "คลิกที่ extension **Smooth Camera**",
            "ar": "الضغط على الملحق **smooth camera*.*"
```

File: tutorials/in-app/cameraParallax.json [hunk 25]
```diff
@@ -173,7 +183,8 @@
             "fr": "Installez l'extension au projet.",
             "es": "Instala la extensión en tu proyecto.",
             "pt": "Instale a extensão no seu projeto.",
-            "th": "ติดตั้ง extension ในโปรเจกต์ของคุณ"
+            "th": "ติดตั้ง extension ในโปรเจกต์ของคุณ",
+			"ar": "تثبيت الملحق في مشروعك."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:187]:
```suggestion
            "th": "ติดตั้ง extension ในโปรเจกต์ของคุณ",
            "ar": "تثبيت الملحق في مشروعك."
```

File: tutorials/in-app/cameraParallax.json [hunk 26]
```diff
@@ -192,7 +203,8 @@
             "fr": "Fermons cette fenêtre.",
             "es": "Cerramos esta ventana.",
             "pt": "Vamos fechar esta janela.",
-            "th": "ปิดหน้าต่างนี้"
+            "th": "ปิดหน้าต่างนี้",
+			"ar": "دعنا نغلق هذا."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:207]:
```suggestion
            "th": "ปิดหน้าต่างนี้",
            "ar": "دعنا نغلق هذا."
```

File: tutorials/in-app/cameraParallax.json [hunk 27]
```diff
@@ -210,7 +222,8 @@
             "fr": "Fermons le menu.",
             "es": "Cerramos el menú.",
             "pt": "Vamos fechar o menu.",
-            "th": "ปิดเมนู"
+            "th": "ปิดเมนู",
+			"ar": "دعنا نغلق هذه القائمة."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:226]:
```suggestion
            "th": "ปิดเมนู",
            "ar": "دعنا نغلق هذه القائمة."
```

File: tutorials/in-app/cameraParallax.json [hunk 28]
```diff
@@ -228,7 +241,8 @@
             "fr": "Ouvrez le panneau des **objets**.",
             "es": "Abre el panel de **objetos**.",
             "pt": "Abra o painel de **objetos**.",
-            "th": "เปิดแผงควบคุม **วัตถุ**"
+            "th": "เปิดแผงควบคุม **วัตถุ**",
+			"ar": "فتح لوحة **الكائنات**."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:245]:
```suggestion
            "th": "เปิดแผงควบคุม **วัตถุ**",
            "ar": "فتح لوحة **الكائنات**."
```

File: tutorials/in-app/cameraParallax.json [hunk 29]
```diff
@@ -248,7 +262,8 @@
             "fr": "Ajoutons maintenant l'extension à notre joueur.\n\nCliquez droit sur **$(player)** ou utilisez le menu à 3 points et sélectionnez **Modifier les comportements**.",
             "es": "Sigamos nuestro jugador con la Cámara con la ayuda de los **comportamientos**.\n\nHaz clic derecho en **$(player)** o usa el menú de 3 puntos y selecciona **Editar comportamientos**.",
             "pt": "Vamos seguir nosso jogador com a Câmera com a ajuda de **comportamentos**.\n\nClique com o botão direito em **$(player)** ou use o menu de 3 pontos e selecione **Editar comportamentos**.",
-            "th": "ทีนี้ เพิ่ม extension ใส่ลงในผู้เล่น\n\nคลิกขวาที่ **$(player)** หรือกดเมนู 3 จุด แล้วเลือก **แก้ไขพฤติกรรม**"
+            "th": "ทีนี้ เพิ่ม extension ใส่ลงในผู้เล่น\n\nคลิกขวาที่ **$(player)** หรือกดเมนู 3 จุด แล้วเลือก **แก้ไขพฤติกรรม**",
+			"ar": "دعنا نصل الكاميرا بشخصيتنا بمساعدة **السلوكيات**:\n\nالضغط على الثلاث نقاط، أو النقر على زر الفأرة الأيمن على الـ **($player)** وتحديد **تحرير السلوكيات**."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:266]:
```suggestion
            "th": "ทีนี้ เพิ่ม extension ใส่ลงในผู้เล่น\n\nคลิกขวาที่ **$(player)** หรือกดเมนู 3 จุด แล้วเลือก **แก้ไขพฤติกรรม**",
            "ar": "دعنا نصل الكاميرا بشخصيتنا بمساعدة **السلوكيات**:\n\nالضغط على الثلاث نقاط، أو النقر على زر الفأرة الأيمن على الـ **($player)** وتحديد **تحرير السلوكيات**."
```

File: tutorials/in-app/cameraParallax.json [hunk 30]
```diff
@@ -257,7 +272,8 @@
             "fr": "Ajoutons maintenant l'extension à notre joueur.\n\nSélectionnez puis appuyez longuement sur **$(player)** et sélectionnez **Modifier les comportements**.",
             "es": "Sigamos nuestro jugador con la Cámara con la ayuda de los **comportamientos**.\n\nSelecciona y pulsa largo **$(player)** y selecciona **Editar comportamientos**.",
             "pt": "Vamos seguir nosso jogador com a Câmera com a ajuda de **comportamentos**.\n\nSelecione e pressione por um longo tempo **$(player)** e selecione **Editar comportamentos**.",
-            "th": "ทีนี้ เพิ่ม extension ใส่ลงในผู้เล่น\n\nเลือกแล้วกดค้างที่ **$(player)** แล้วเลือก **แก้ไขพฤติกรรม**"
+            "th": "ทีนี้ เพิ่ม extension ใส่ลงในผู้เล่น\n\nเลือกแล้วกดค้างที่ **$(player)** แล้วเลือก **แก้ไขพฤติกรรม**",
+			"ar": "دعنا نصل الكاميرا بشخصيتنا بمساعدة **السلوكيات**:\n\nتحديد، ثم ضغطة مطولة على الـ **($player)** وتحديد **تحرير السلوكيات**."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:276]:
```suggestion
            "th": "ทีนี้ เพิ่ม extension ใส่ลงในผู้เล่น\n\nเลือกแล้วกดค้างที่ **$(player)** แล้วเลือก **แก้ไขพฤติกรรม**",
            "ar": "دعنا نصل الكاميرا بشخصيتنا بمساعدة **السلوكيات**:\n\nتحديد، ثم ضغطة مطولة على الـ **($player)** وتحديد **تحرير السلوكيات**."
```

File: tutorials/in-app/cameraParallax.json [hunk 31]
```diff
@@ -274,7 +290,8 @@
             "fr": "Les **comportements** de **l'objet** se trouvent dans cet onglet.",
             "es": "Los **comportamientos** del **objeto** se encuentran en esta pestaña.",
             "pt": "Os **comportamentos** do **objeto** estão nesta guia.",
-            "th": "ดู **พฤติกรรม** ของ **วัตถุ** ที่นี่"
+            "th": "ดู **พฤติกรรม** ของ **วัตถุ** ที่นี่",
+			"ar": "رؤية **سلوكيات الكائن** تأتي من هنا."
```
Comment (Bouh) [inline tutorials/in-app/cameraParallax.json:294]:
```suggestion
            "th": "ดู **พฤติกรรม** ของ **วัตถุ** ที่นี่",
            "ar": "رؤية **سلوكيات الكائن** تأتي من هنا."
```

**Comment by Bouh** [comment]
Hello, thanks for the new Arabic language.
I fixed some unexpected indentations in front of some new lines, and 2 small error.

@ClementPasteau 
It look good to me!

**Comment by EyadMahm0ud** [comment]
> Hello, thanks for the new Arabic language. I fixed some unexpected indentations in front of some new lines, and 2 small error.
> 
> @ClementPasteau It look good to me!

I see, for some reason my editor is playing around with font size (even Thai), I'll fix it by adding extra space.. Thanks for reviewing and correcting my code @Bouh <3

**Comment by ClementPasteau** [comment]
Thanks for your contribution @EyadMahm0ud!

I'll merge it and it will be deployed in the next few hours

**Pull request merged**